hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/metrics/RBFMetrics.java

```diff
@@ -783,6 +784,23 @@ private long getNameserviceAggregatedLong(ToLongFunction<MembershipStats> f) {
     }
   }
 
+  private BigInteger getNameserviceAggregatedBigInt(
+      ToLongFunction<MembershipStats> f) {
+    try {
+      List<MembershipState> states = getActiveNamenodeRegistrations();
+      BigInteger sum = new BigInteger("0");
+      for (MembershipState state : states) {
+        long lvalue = f.applyAsLong(state.getStats());
+        BigInteger bvalue = new BigInteger(String.valueOf(lvalue));
```

Comment:
String is not needed.
```suggestion
        BigInteger bvalue = BigInteger.valueOf(lvalue);
```

In addition, `new BigInteger("0")` can be replaced with `BigInteger.valueOf(0)`.